### ISSUE TITLE
Add video player to Audacity

### ISSUE THREAD

**Issue by nnethery** [audacity/audacity#4875]
### Your idea

I see Audacity as a great tool to be used in the film/video context for sound editors and designers. Currently, Audacity extracts the audio from a video clip when a video is imported.

### Problem to be solved

Sound editors need a way to synchronize their audio timeline to a video and I think this starts with building a floating media player that is synchronized to the audio tracks timeline. This provides the basis for audio/video work done directly in Audacity.

### Additional context

Since Audacity is moving away from wxWidgets to Qt, I think it would be wise to work off of the `qt` branch and use Qt widgets for this. I'm happy to personally contribute to this. Since this is a complex feature I can break down into steps/concerns like the following if there is interest in this:
* Syncing with scrubbing
* Allowing a media track to be moved along the timeline
* Frame rate to sample rate conversion
* Aligning video/audio when a clip is trimmed
* Import/export UX

**Comment by LWinterberg** [comment]
Thank you for your interest! Unfortunately, I don't think we're ready for this just yet; a massive refactoring is currently ongoing and the Qt branch acts only as a UI component demo right now. 

What you can do right now though is see if you can get Audacity's mod-script-pipe talk to a thing using libvlc or something similar though. While probably being very limited, it could be useful immediately and wouldn't be blocked by our refactors. 

I'll close this issue as a duplicate of #1023 which I have tracked elsewhere and will dust off once the right time has come.
